osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -572,6 +572,30 @@
           "access": "RW"
         }
       ]
+    },
+    {
+      "tag": "ALARM_FILTER_1",
+      "description": "Alarm filter 1",
+      "note": null,
+      "class-id": 1,
+      "version": 0,
+      "obis": "0.0.97.98.10.255",
+      "group": "ELECTRICITY",
+      "meterTypes": [
+        "SP",
+        "PP"
+      ],
+      "properties": {},
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "double-long-unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
```

Comment:
Ik zie in de spec geen 0 waarde staan

Reply:
Nee, ik heb dat hetzelfde gehouden als bij die Alarm Register, maar ik zal nog wel voor de zekerheid even aan Stefan vragen.

Reply:
Nee, maar dat staat er bij die Alarm register ook niet en zoals ik het begrijp moet je er dezelfde waarden in kunnen zetten. Anders is die ALARM_REGISTER_1 ook niet goed.

Reply:
Ik denk dat ook bij AlarmRegister de value niet 0 moet zijn idd

Reply:
Ik heb dit even met Stefan overlegd omdat ik even de weg kwijt raakte hoe dit nu precies zit met die value. De value wordt nu nog niet gebruikt, maar wordt pas relevant als deze bijvoorbeeld in de simulator gebruikt wordt. Voor nu is daarom de 0 prima. Eigenlijk staat er dan voor nu dus dat er geen alarm in het filter zit en kan dit later aangepast worden.

Reply:
Ik ben het er niet mee eens, ik denk dat als we objecten toe voegen aan de configuratie dat deze moet kloppen volgens de specificatie........ We kunnen niet zomaar 0 invullen. Dan moeten we straks nog een keer in de spec duiken